client/landing/gutenboarding/constants.ts

```diff
@@ -9,6 +9,9 @@ import type { ValuesType } from 'utility-types';
  */
 export const FLOW_ID = 'gutenboarding';
 
+export const DOMAINS_SUGGESTION_COUNT_IN_POPOVER = 5;
+export const DOMAINS_SUGGESTION_COUNT_IN_MODAL = 10;
```

Comment:
I'm not sure these new constants are needed in Gutenboarding.

We already know we want a single free domain and there is `PAID_DOMAINS_TO_SHOW` defined in this file. 

Then there is the bigger value (`DOMAINS_SUGGESTION_COUNT_IN_MODAL`) which should be handled internally by the DomainPicker package. According to latest changes, even the Modal will display 6 items by default and then just double that amount.